src/Libraries/RevitNodes/Elements/FamilyType.cs

```diff
@@ -60,6 +65,29 @@ private void InitFamilySymbol(Autodesk.Revit.DB.FamilySymbol symbol)
             InternalSetFamilySymbol(symbol);
         }
 
+        /// <summary>
+        /// Get Solid from Element helper method
+        /// </summary>
+        /// <param name="element"></param>
+        /// <returns></returns>
+        private static Solid GetSolidFromElement(Autodesk.Revit.DB.Element element)
+        {
+            GeometryElement geo = element.get_Geometry(new Options() { ComputeReferences = true });
+
+            // Get geometry instance from geometry element
+            var enumerator = geo.GetEnumerator();
+            enumerator.MoveNext();
+            GeometryInstance i = (GeometryInstance)enumerator.Current;
+
+            // Get solid from geometry instance
+            var geom2 = i.GetInstanceGeometry();
+            var enumarator2 = geom2.GetEnumerator();
```

Comment:
yes, only required the the first geometryinstance & solid at this point because all of the imported should be in there.